probert/network.py

```diff
@@ -24,7 +24,11 @@
 
 import pyudev
 
-from probert import _nl80211, _rtnetlink
+try:
+    from probert import _nl80211, _rtnetlink
+except ImportError:
+    pass
```

Comment:
Let;s log the error:

except ImportError as e:
   LOG.warning('Failed import network library modules: %s', e)

Reply:
thanks! added